conf/config.json

```diff
@@ -40,31 +32,16 @@
                 }
             },
             "destination": {
-                "s3": {
-                    "hosts": [
-                        {
-                            "host": "127.0.0.2",
-                            "port": 9000
-                        }
-                    ],
-                    "transport": "https"
-                },
-                "auth": {
-                    "type": "account",
-                    "account": "lisa",
-                    "vault": {
-                        "hosts": [
-                            {
-                                "host": "127.0.0.2",
-                                "port": 9500
-                            }
-                        ]
-                    }
-                },
+                "transport": "https",
+                "bootstrapList": ["127.0.0.1:8443"],
```

Comment:
This looks ok for enterprise, but we may want to keep the possibility to configure S3 and vault separately, especially for open source zenko which users may not want to install nginx as a requirement?